SpatialGDK/Documentation/features.md

```diff
@@ -48,6 +46,10 @@ Actor property handover is a new feature we’re introducing with the SpatialOS
 
 `Singleton Actors` is a new term that describes Actors which contain global state and logic.  Only one of these Actors exists in a game world. The SpatialOS Unreal GDK supports both client accessible and server only singletons (respectively known as public and private singletons). For more information see the documentation on [Singleton Actors]({{urlRoot}}/content/singleton-actors).
 
+## Cross-server RPCs
+
+`Cross-server RPCs` is a new type of RPC that functions between two servers. These will enable a server without authority over an Actor to communicate with the server which has authority and is able to manipulate the Actor. For more information, see [Cross-server RPC]({{urlRoot}}/content/dummy-link) documentation.
```

Comment:
The link will be updated once doc is ready. @danielimprobable will work on the doc.

Reply:
The linting is fierce! It won't let you put this non-link in. Suggest adding this as a "TODO" comment and I will add a note to self to add this.
So:
"[//]: # (TODO: Add link to the Cross-server RPC doc)"
I have added a TODO note in the Doc of Docs Toc master list.